### PR TITLE
CONCF-676 | As a developer I would like to have clean code related with videos in Mercury so I can add changes faster

### PR DESCRIPTION
**Pull request by tomasznapieralski**

### CODE REVIEW

**Comment by tomasznapieralski** [comment]
@Warkot @hakubo @bognix

File: front/scripts/main/models/MediaModel.ts [hunk 1]
```diff
@@ -21,6 +21,16 @@ interface LightboxMediaRefs {
 }
 
 App.MediaModel = Em.Object.extend({
+
+	/**
+	 * In order to have consistency in input data we are wrapping them into array if they are not
+	 */
+	init: function () {
```
Comment (Warkot) [inline front/scripts/main/models/MediaModel.ts]: Return type `: void`.

File: front/scripts/main/models/MediaModel.ts [hunk 2]
```diff
@@ -21,6 +21,16 @@ interface LightboxMediaRefs {
...
+		if (! Ember.isArray(this.get('media'))) {
```
Comment (Warkot) [inline front/scripts/main/models/MediaModel.ts]: Space after `!` is excessive.

File: front/scripts/main/models/MediaModel.ts [hunk 3]
```diff
@@ -21,6 +21,16 @@ interface LightboxMediaRefs {
...
+			this.set('media', [this.get('media')]);
```
Comment (Warkot) [inline front/scripts/main/models/MediaModel.ts]: It's the second time you're using getter for `media` in the same context — you could get it once and keep it in a variable.

File: front/scripts/main/components/TrendingVideosComponent.ts [hunk 4]
```diff
@@ -7,7 +7,7 @@ App.TrendingVideosComponent = Em.Component.extend({
 	actions: {
 		openLightbox: function (video: any): void {
 			var mediaModel = App.MediaModel.create({
-				media: [video]
+				media: video,
```
Comment (Warkot) [inline front/scripts/main/components/TrendingVideosComponent.ts]: Is trailing comma OK in our coding conventions? AFAIK we shouldn't use it.

**Comment by bognix** [comment]
LGTM

**Comment by vforge** [comment]
LGTM

**Pull request merged**